### ISSUE TITLE
Review Process after user submits new hypocrite

### ISSUE THREAD

**Issue by AndrewGoldstein** [AndrewGoldstein/50-hypocrites#3]

**Comment by stonecrafter** [comment]
Seems like this would be part of the admin mode/panel (requires #6 to be done first)? Assuming the reviewers will be admins.

**Comment by AndrewGoldstein** [comment]
@fromtheothershore agreed, we'll have to get the admin mode set up first - perhaps set up a login system?
